File: manual/code/generate_strategies/RA(name)/java/java/function_asyncio_run.java
package com.example;

import reactor.core.publisher.Mono;
import java.time.Duration;

public class AsyncTaskHandler {
    public String run_async_task(String name, int delay) {
        class AsyncTaskHandler {
            public String run_async_task(String name, int delay) {
                return Mono.delay(Duration.ofSeconds(delay))
                        .map(tick -> String.format("Task %s completed after %d seconds", name, delay))
                        .block();
            }
        }
        return new AsyncTaskHandler().run_async_task(name, delay);
    }
}

File: data/python/src/function_asyncio_run.py
import asyncio

class AsyncTaskHandler:
    def run_async_task(self, name: str, delay: int) -> str:
        
        async def async_task(name: str, delay: int) -> str:
            await asyncio.sleep(delay)
            return f"Task {name} completed after {delay} seconds"

        return asyncio.run(async_task(name, delay))
